Test/Program.cs

```diff
@@ -164,6 +166,60 @@ private static void PrintFileInfo(string file, string outputDirectory, Options o
                 Console.WriteLine();
             }
         }
+        
+        /// <summary>
+        /// Helper method to check for .WISE section executable.
+        /// </summary>
+        /// <param name="stream">Stream that represents the extractable data</param>
+        /// <param name="file">File path</param>
+        /// <param name="outputDirectory">Output directory path</param>
+        /// <param name="options">User-defined options</param>
+        private static void WiseSectionHelper(Stream stream, string outputDirectory, Options options, string file)
```

Comment:
Please move this helper under the `Extraction` region below.
Please also rename it to something like `TryExtractWiseSection`.